rm-datasource/src/main/java/io/seata/rm/datasource/undo/parser/JacksonUndoLogParser.java

```diff
@@ -78,4 +105,46 @@ public BranchUndoLog decode(byte[] bytes) {
         }
     }
 
+    private static class TimestampSerializer extends JsonSerializer<Timestamp> {
+
+        @Override
+        public void serializeWithType(Timestamp timestamp, JsonGenerator gen, SerializerProvider serializers, TypeSerializer typeSerializer) throws IOException {
+            gen.setCurrentValue(timestamp);
```

Comment:
is necessary？

Reply:
I will try to check the api to think is it necessary.